### PR TITLE
Style mechanics article

### PR DESCRIPTION
**Pull request by Ludmila-N**
Minor comma/style changes to contributor/04-mechanics-of-contributing-article

### CODE REVIEW

File: contributor/04-mechanics-of-contributing-article.asciidoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ It is very likely that, as you do this, everything will feel more and more natur
 
 One key difference is the turnaround time.
 With every first time contribution you are coming to a new (host) team.
-As a result, you'll need to get to know their code base and the technology used, but also their preferred development environment (think test framework, build system).
+As a result, you'll need to get to know their code base and the technology used, and also their preferred development environment (think test framework, build system).
```
Comment (rrrutledge) [inline contributor/04-mechanics-of-contributing-article.asciidoc]:
```suggestion
As a result, you'll need to get to know their code base, technologies used, and also their preferred development environment (think test framework, build system).
```

File: contributor/04-mechanics-of-contributing-article.asciidoc [hunk 2]
```diff
@@ -71,10 +71,10 @@ This type of high-level, early upfront agreement will save time in rework or rej
 
 ==== Communication and unblocking yourself
 
-Great, you've made yourself familiar with the host team's approach and they are looking forward to receive your pull request.
+Great, you've made yourself familiar with the host team's approach, and they are looking forward to receive your pull request.
 Which gotchas are there waiting for you now?
 
-First, you'll be in less direct contact with them and second, you aren't expected to be as knowledgeable and proficient as you might be on the full-time projects that your team owns.
+First, you'll be in less direct contact with them, and second, you aren't expected to be as knowledgeable and proficient as you might be on the full-time projects that your team owns.
```
Comment (rrrutledge) [inline contributor/04-mechanics-of-contributing-article.asciidoc]:
```suggestion
First, you'll be in less direct contact with them.  Second, you aren't expected to be as knowledgeable and proficient as you might be on the full-time projects that your team owns.
```

File: contributor/04-mechanics-of-contributing-article.asciidoc [hunk 3]
```diff
@@ -118,34 +118,34 @@ This might actually be pretty similar to what you may already be doing on your o
...
- ** If you have a larger feature that you are contributing, it often helps to split it in multiple pull requests that get submitted, reviewed and accepted sequentially.
+ ** If you have a larger feature that you are contributing, it often helps to split it in multiple pull requests that are submitted, reviewed and accepted sequentially.
 You can still bind them together with an issue that you are referring to.
-  *** Some tools also have Draft / WIP pull request functionality that you can use to explicitly mark unfinished and non-polished work and still get early feedback from your host team's https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/01-introduction.asciidoc[_Trusted Committers_].
+  *** Some tools also have Draft / WIP pull request functionality that you can use to explicitly mark unfinished and non-polished work and still get early feedback from your host team's https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/02-ensuring-product-quality.asciidoc[_Trusted Committers_].
```
Comment (rrrutledge) [inline contributor/04-mechanics-of-contributing-article.asciidoc]: I'm not clear why the change to this article?

Comment (Ludmila-N) [reply contributor/04-mechanics-of-contributing-article.asciidoc]: I had gone back and forth between keeping the links pointing to the same place, and providing content relevant reference points.
I remember you suggesting we link to specific articles vs always to the intro, but I could never find a passage specific enough to warrant pointing to a non-intro article.
This portion talks about the TCs in light of their responsibilities towards contributors, and I felt like it is an opportunity to link to the relevant article.
With that, I also agree that this is not obvious for someone to click on after they clicked on other TC links, so happy to revert if we all decide towards consistency.

Comment (rrrutledge) [reply contributor/04-mechanics-of-contributing-article.asciidoc]: Seems fine to me.
